src/core/placeTooltip.js

```diff
@@ -206,7 +203,7 @@ export default function placeTooltip(
     tooltipCssClass = this._options.tooltipClass;
   }
 
-  tooltipLayer.className = `introjs-tooltip ${tooltipCssClass}`.replace(
+  tooltipLayer.className = `introjs-tooltip ${tooltipCssClass || ''}`.replace(
```

Comment:
I would prefer a simple array with filter(Boolean) and join(' '):

```js
tooltipLayer.className = [
  'introjs-tooltip',
  tooltipCssClass,
].filter(Boolean).join(' ');
```

Reply:
@bozdoz done. also added two tests for this specific line.